build/building/cache/garbage-collection.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Garbage collection
+keywords: build, buildx, buildkit, garbage collection, prune
+---
+
+While [`docker build prune`](../../../engine/reference/commandline/builder_prune.md)
+or [`docker buildx prune`](../../../engine/reference/commandline/buildx_prune.md)
+commands run at once, garbage collection runs periodically and follows an
+ordered list of prune policies.
+
```

Comment:
I think it would be nice to give a little bit more context about where the GC runs and how it works before delving into config instructions.

```suggestion

Garbage collection runs in the BuildKit daemon. The daemon clears the build cache when the cache size becomes too big, or when the cache age expires. The following sections describe how you can configure both the size and age parameters by defining garbage collection policies.

```